pkg/interfaces/handler/nope_handler.go

```diff
@@ -0,0 +1,131 @@
+//go:generate mockgen -source=$GOFILE -destination=mock_$GOPACKAGE/mock_$GOFILE
+package handler
+
+import (
+	"context"
+	"encoding/json"
+	"log"
+	"net/http"
+	"time"
+
+	"github.com/satorunooshie/swipe-shukatu/pkg/dcontext"
+	nopeM "github.com/satorunooshie/swipe-shukatu/pkg/domain/model"
+	nopeU "github.com/satorunooshie/swipe-shukatu/pkg/usecase"
+)
+
+type NopeHandler interface {
+	HandleSelect() http.HandlerFunc
+	HandleInsert() http.HandlerFunc
+	HandleUpdate() http.HandlerFunc
+	HandleDelete() http.HandlerFunc
+}
+
+type nopeHandler struct {
+	nopeUseCase nopeU.NopeUseCase
+}
+
+// NewNopeHandler
+func NewNopeHandler(nopeU nopeU.NopeUseCase) NopeHandler {
+	return &nopeHandler{
+		nopeUseCase: nopeU,
+	}
+}
+
+// HandleSelect
+func (nopeH *nopeHandler) HandleSelect() http.HandlerFunc {
+	return func(writer http.ResponseWriter, request *http.Request) {
+		ctx := request.Context()
+		ctx, cancel := context.WithTimeout(ctx, 10*time.Second)
+		defer cancel()
+		UID := dcontext.GetUIDFromContext(ctx)
+		nopes, err := nopeH.nopeUseCase.Select(ctx, UID)
+		if err != nil {
+			log.Printf("[ERROR] failed to Select from nope: %v", err.Error())
```

Comment:
```suggestion
			log.Printf("[ERROR] failed to select from nope: %v", err.Error())
```
このエラーのところずっと気になってたんだけど、なんで大文字？？
エラー箇所を特定したいならpackage.Methodとかmethod.methodとかが良さそう